Fabric.Identity.API/scripts/Install-Identity-Windows.ps1

```diff
@@ -102,10 +102,27 @@ if((Test-Path $zipPackage))
 }
 
 
-if(!(Test-Prerequisite '*.NET Core*Windows Server Hosting*' 1.1.30327.81))
+if(!(Test-Prerequisite-Exact '*.NET Core*Windows Server Hosting*' 1.1.30327.81))
 {
-    Write-Host ".NET Core Windows Server Hosting Bundle minimum version 1.1.30327.81 not installed...download and install from https://go.microsoft.com/fwlink/?linkid=844461. Halting installation."
-    exit 1
+    try{
+		Write-Host "Windows Server Hosting Bundle version 1.1.30327.81 not installed...installing version 1.1.30327.81"
+        Write-Host "downloading to:" $env:Temp
+		Invoke-WebRequest -Uri https://go.microsoft.com/fwlink/?linkid=844461 -OutFile $env:Temp\bundle.exe
+		Start-Process $env:Temp\bundle.exe -Wait -ArgumentList '/quiet /install'
+		net stop was /y
+		net start w3svc			
+	}catch{
+	    Write-Error "Could not install .NET Windows Server Hosting bundle. Please install the hosting bundle before proceeding. https://go.microsoft.com/fwlink/?linkid=844461"
```

Comment:
Do we know if this error will halt the script? Sometimes certain exceptions don't cause powershell to stop, and in this case we want the script to stop so the user can resolve the reported error.

Reply:
The next line that throws the exception will halt the script. I can make the script a little smarter to let the user know if there are other versions installed and if that is not the cause of the error than just display the error message without throwing the exception. I will use ErrorAction -Stop to halt the script